Firmware/actions.py

```diff
@@ -1,7 +1,4 @@
-import usb_hid
 import time
-import supervisor
```

Comment:
I don't know if it is actually save to delete these here. I needed to remove them to run the test code on my host machine. Removing them didn't see to break anything but that might not be the case on the real keyboard and needs to be tested/confirmed